.github/workflows/flatpak.yml

```diff
@@ -32,54 +32,36 @@ on:
 name: Flatpak
 
 jobs:
-  build:
-    name: Build the app
-    runs-on: ubuntu-20.04
+  flatpak:
+    name: Flatpak
+    runs-on: ubuntu-latest
     container:
-      image: nahuelwexd/replay-ci-image:latest
+      image: bilelmoussaoui/flatpak-github-actions:gnome-nightly
       options: --privileged
+    strategy:
+      matrix:
+        arch: [x86_64, aarch64]
+      fail-fast: false
     steps:
       - name: Checkout
         uses: actions/checkout@v2.3.4
 
-      - name: Cache dependencies
-        uses: actions/cache@v2.1.6
-        with:
-          path: .flatpak-builder
-          key: ${{ runner.os }}-flatpak-${{ hashFiles('**/com.github.nahuelwexd.Replay.yml') }}
-          restore-keys: |
-            ${{ runner.os }}-flatpak-
-
-      - name: Build
+      - name: Install deps
         run: |
-          flatpak-builder --repo=repo --install-deps-from=gnome-nightly flatpak_app \
-              build-aux/flatpak/com.github.nahuelwexd.Replay.yml
+          dnf install -y docker
```

Comment:
Uh, this isn't an error? ur running `dnf` on an ubuntu image :thinking:

Reply:
I think it can be fixed with `apt install -y docker`

Reply:
Not really, all this job runs on a docker image, as you can see in the part that says "container"

The docker image is based on Fedora, as documented [here](https://github.com/bilelmoussaoui/flatpak-github-actions#docker-image).

So using dnf is the right thing to do, plus the CI logs confirm it.